### PR TITLE
feat(swc_plugin_import): support camelCase and snake_case

### PR DESCRIPTION
**Pull request by SoonIter**
## Summary

support camelCase and snake_case

related rsbuild issue https://github.com/web-infra-dev/rsbuild/issues/1216?notification_referrer_id=NT_kwDOBLvAAbM5MDE0MjIxMjQwOjc5NDEzMjQ5#event-11426342492

## Require Documentation?

- [x] No
- [ ] Yes, the corresponding rspack-website PR is \_\_

### CODE REVIEW

**Comment by h-a-n-a** [comment]
Would you please also add a test case for `camelCase`?

File: crates/swc_plugin_import/tests/fixtures/custom-name-tpl-camel/option.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "pluginImport": [
       {
         "libraryName": "foo",
-        "customName": "foo/__custom_es__/{{kebabCase member }}"
+        "customName": "foo/__custom_es__/{{ camelCase member }}"
```
Comment (h-a-n-a) [inline crates/swc_plugin_import/tests/fixtures/custom-name-tpl-camel/option.json:6]: Thanks, but I don't think these cases are working anymore here. They are now one of builtin-swc-loader's feature, tests of which are located here `packages/rspack/tests/configCases/builtin-swc-loader/plugin-import`.

Comment (SoonIter) [reply crates/swc_plugin_import/tests/fixtures/custom-name-tpl-camel/option.json:6]: OK, I plan to move all the `swc_plugin_import` test cases to `configCases/builtin-swc-loader/plugin-import-cases`

**Review by h-a-n-a — APPROVED** [review]
Thank you

**Pull request merged**